packages/vscode/README.md

```diff
@@ -1,3 +1,26 @@
-# WEBlsp's vscode extension
+<picture>
```

Comment:
This README will show in VS Code directly, so there's some concerns to be had regarding how it gets formatted there, and what are the expectations of that community.

For instance, it's not useful to link the repo, because in VS Code the repository is linked on the side. Same thing for issues.

The installation is also not necessarily useful, as in the context of VS Code, it's obvious how to install.